meal-planner/src/App.js

```diff
@@ -4,41 +4,123 @@ import woodBackground from './assets/wood_background.jpg'
+        {
+            weekday: 'Mittwoch',
+            id: uuidv4(),
+            meal: '',
+            instructions: '',
+            ingredients: '',
+            image: '',
+        },
+        {
+            weekday: 'Donnerstag',
+            id: uuidv4(),
+            meal: '',
+            instructions: '',
+            ingredients: '',
+            image: '',
+        },
+        {
+            weekday: 'Freitag',
+            id: uuidv4(),
+            meal: '',
+            instructions: '',
+            ingredients: '',
+            image: '',
+        },
+        {
+            weekday: 'Samstag',
+            id: uuidv4(),
+            meal: '',
+            instructions: '',
+            ingredients: '',
+            image: '',
+        },
+        {
+            weekday: 'Sonntag',
+            id: uuidv4(),
+            meal: '',
+            instructions: '',
+            ingredients: '',
+            image: '',
+        },
     ]
 
     const [weekdays, setWeekdays] = useState(mealPlan)
-
     const [selectedWeekday, setSelectedWeekday] = useState('')
+    const [showRecipe, setShowRecipe] = useState(false)
+
+    const [selectedMealTitle, setSelectdedMealTitle] = useState('')
+    const [selectedMealInstructions, setSelectedMealInstructions] = useState('')
+    const [selectedMealIngredients, setSelectedMealingredients] = useState('')
+    const [selectedImage, setSelectedImage] = useState('')
 
     function handleClick(event) {
         setSelectedWeekday(event.currentTarget.id)
     }
 
-    function selectMeal(meal, id) {
+    function selectMeal(meal, id, instructions, ingredients, image) {
         setSelectedWeekday('')
         const updatedWeekdays = weekdays.map((weekday) =>
-            weekday.id === id ? { ...weekday, meal } : weekday
+            weekday.id === id
+                ? { ...weekday, meal, ingredients, instructions, image }
+                : weekday
         )
         setWeekdays(updatedWeekdays)
     }
 
+    //button to show recipe
+    function handleShowRecipe(meal, ingredients, instructions, image) {
+        setShowRecipe(true)
+        setSelectdedMealTitle(meal)
+        setSelectedMealingredients(ingredients)
+        setSelectedImage(image)
+        setSelectedMealInstructions(instructions)
+    }
+
+    //closing box on recipe page
+    function handleCloseRecipe() {
+        setShowRecipe(false)
+    }
+
     return (
         <AppStyled>
             <header>
                 <img src={logo} alt="" />
             </header>
             <section>
+                {showRecipe ? (
+                    <RecipePage
+                        closeRecipe={handleCloseRecipe}
+                        title={selectedMealTitle}
+                        image={selectedImage}
+                        ingredients={selectedMealIngredients}
+                        instructions={selectedMealInstructions}
+                    />
+                ) : (
+                    ''
+                )}
```

Comment:
Its rendering the RecipePage part above your normal page. Its not visible, but it's in the HTML and if you test it, there is still the scrollbar slightly visible from the page behind. Would be better if you render it **instead** of the normal page.
So maybe put the other part in place where the actual empty string is? 
Than the RecipePage is rendered in case of true, or the normal view if false.